src/index.js

```diff
@@ -115,23 +109,26 @@ module.exports = function(babel) {
 
             removals.push(specifierPath);
 
-            // Repalce the node with a new `var name = Ember.something`
-            replacements.push(
-              t.variableDeclaration('var', [
-                t.variableDeclarator(
-                  local,
-                  t.memberExpression(t.identifier('Ember'), t.identifier(global))
-                ),
-              ])
-            );
+            // Replace the occurences of the imported name with the global name.
+            replacements.push([
+              local.name,
+              `Ember.${global}`,
```

Comment:
I'd rather we just use the value in the mapping for `global` (we strip `Ember.` in the reverseMapping build up, then bring it back here).